extension/icu/icu-extension.cpp

```diff
@@ -145,6 +245,20 @@ void ICUExtension::Load(DuckDB &db) {
 	CreateScalarFunctionInfo sort_key_info(move(sort_key));
 	catalog.CreateFunction(*con.context, &sort_key_info);
 
+	// Time Zones
+	auto &config = DBConfig::GetConfig(*db.instance);
+	config.AddExtensionOption("TimeZone", "The current time zone", LogicalType::VARCHAR, SetICUTimeZone);
+	Value utc("UTC");
```

Comment:
Does ICU also have the capability of detecting the users' current time zone? Perhaps that would be a more sensible default than UTC.

Reply:
It would, butI wasn't sure how reliable it was on all platforms. I can change it though so we can get bug reports ;-)

Reply:
I would hope ICU is at least semi-well tested, considering how widely used it is :)

Reply:
I meant DuckDB bug reports/feature requests(!) But I'll see what I can do. The one problem is you can't really test it.

Reply:
Seems to work on my Mac.